packages/www/docs/guides/upload-a-video/index.mdx

```diff
@@ -0,0 +1,775 @@
+---
+title: Upload a Video
+description: Livepeer's Upload a Video Guide
+metaTitle: Upload a Video Guide - Livepeer.com
+metaDescription: Livepeer's Upload a Video Guide
+---
+
+# Upload a Video
+
+This tutorial covers how to upload the video to Livepeer to allow viewers to
+playback stored video assets in your application.
+
+## Caveats
+
+- Have to manually check the status of the file using `List Task` method to see
+  if it has finished uploading.
+- File size is limited to 1GB. Any files greater than 1GB will error with
+  NETWORK CHANGED.
+- Only MP4 files encoded with H.264 and AAC are supported.
+
+## Installation
+
+- Install [Node.js](https://nodejs.org/en/)
+- Install package manager with
+  [NPM](https://docs.npmjs.com/cli/v8/configuring-npm/install) or
+  [Yarn](https://classic.yarnpkg.com/en/docs/install#mac-stable)
+- Install [Axios](https://axios-http.com/)
+  - Using npm
+  ```bash
+  npm axios
+  ```
+  - Using yarn
+  ```bash
+  yarn add axios
+  ```
+- Get a
+  [Livepeer Video Service API key](https://livepeer.com/docs/guides/start-live-streaming/api-key)
+
+## Assets
+
+There are two methods to upload a video to Livepeer:
+
+- Upload via URL, with an `.mp4` file format
+
+- Upload locally
```

Comment:
We only support MP4 everywhere, and the caveat has already been made in the beginning. No need to reemphasize it here and only in one of the methods.

Can also improve the wording:
```suggestion
- Import from a URL

- Directly upload from a local file
```
(we theoretically wanted to use "upload" on both cases but I don't really think it makes sense... The API is still called `/import` anyway so it will be confusing not to use that here. @pglowacky do you really think we need to call it upload here? If so let me know and I can rename the API as well)